sdks/java/extensions/sorter/README.md

```diff
@@ -18,11 +18,11 @@
 -->
 
 #Sorter
-This module provides the SortValues transform, which takes a `PCollection<KV<K,Iterable<V>>> and produces a PCollection<KV<K,Iterable<V>>>` where the `Iterable<V>` has been sorted for each key. It will efficiently and scalably sort the iterables, even if they are large (do not fit in memory).
+This module provides the SortValues transform, which takes a `PCollection<KV<K, Iterable<KV<K2, V>>>>` and produces a `PCollection<KV<K, Iterable<KV<K2, V>>>>` where the `Iterable<<KV<K2, V>>>` has been sorted for each secondary key (`K2`). It will efficiently and scalably sort the iterables, even if they are large (do not fit in memory).
```

Comment:
Think a bit about the language here to as clearly as possibly represent the exact semantics. But, I think it's something like this:

where for each key `K`, the paired `Iterable<KV<K2, V>>` has been sorted by the byte-encoding of `K2`.

Is that correct? Is the byte-encoding of `V` included as a tiebreaker if two `K2` are equal?